recipes/thorvg/all/conanfile.py

```diff
@@ -77,6 +80,8 @@ def _compilers_minimum_version(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if Version(self.version) > "0.13.0":
```

Comment:
But does the `with_simd` still have an effect in old versions? What happens if their values differs in versions where both exist?

Reply:
The thing is that it does not exist in the old versions. Perhaps, it is not clear in the changelog 😓
In summary:

* Versions < 0.13.0 -> `vector`
* Versions >= 0.13.1 -> ~`vector`~ `simd`

Reply:
See here https://github.com/thorvg/thorvg/compare/v0.13.0...v0.13.1#diff-76ea9078099f2ee8cc0e760a874dd153f225ba16ad10375c24ab6cd53518f3c4L24